lib/configure.js

```diff
@@ -254,18 +254,21 @@ function configure (gyp, argv, callback) {
     //  - the out/Debug directory
     //  - the root directory
     var node_exp_file = undefined
-    if (process.platform === 'aix') {
+    if (process.platform === 'aix' || process.platform === 'os390') {
+      var ext = process.platform === 'aix' ? 'exp' : 'x'
       var node_root_dir = findNodeDirectory()
-      var candidates = ['include/node/node.exp',
-                        'out/Release/node.exp',
-                        'out/Debug/node.exp',
-                        'node.exp']
+      var candidates = ['include/node/node',
+                        'out/Release/node',
+                        'out/Debug/node',
+                        'node'].map((file) => {
```

Comment:
You can't use arrow functions just yet, they don't parse in old node.js versions.

Reply:
Fixed